### PR TITLE
openldap: update to 2.6.3.

### PR DESCRIPTION
**Pull request by Emru1**
OpenLDAP update to 2.6.3

This update changes naming of libraries, from `libldap-2.4.so.2` to `libldap.so.2`, so it demanded changes in `common/shlibs`
Also, `libldap_r` was deprecated and merged with `libldap` [https://lists.openldap.org/hyperkitty/list/openldap-announce@openldap.org/thread/BH3VDPG6IYYF5L5U6LZGHHKMJY5HFA3L/](url)

I've built new version and checked build status on x86_64 for those packages which were given by `xrevshlib` as dependent on `openldap` or `libldap`.

I've never made PR this big to void-packages, please, check if everything with this PR is ok

### CODE REVIEW

**Comment by Emru1** [comment]
is there any problem with this PR?

File: srcpkgs/python3-ldap/template [hunk 1]
```diff
@@ -16,5 +16,5 @@ distfiles="${PYPI_SITE}/p/${_pkgname}/${_pkgname}-${version}.tar.gz"
 checksum=7d1c4b15375a533564aad3d3deade789221e450052b21ebb9720fb822eccdb8e
 
 pre_build() {
-	sed -i "s:^libs = .*:libs = ldap_r lber sasl2 ssl crypto:" setup.cfg
+	sed -i "s:^libs = .*:libs = ldap lber sasl2 ssl crypto:" setup.cfg
```
Comment (classabbyamp) [inline srcpkgs/python3-ldap/template:19]: make this a vsed

File: srcpkgs/openldap/template [hunk 2]
```diff
@@ -12,17 +12,18 @@ configure_args="--prefix=/usr
...
  --enable-syslog --enable-slapd --enable-crypt --enable-spasswd
  --enable-bdb --enable-hdb --enable-overlays=mod --enable-meta=mod
- --disable-static --with-yielding_select=yes --enable-modules"
+ --disable-static --with-yielding_select=yes --enable-modules
+ --enable-balancer=mod"
 hostmakedepends="pkg-config groff"
-makedepends="openssl-devel libsasl-devel db-devel libltdl-devel"
-depends="openldap-tools>=${version}_${revision}"
+makedepends="openssl-devel libsasl-devel db-devel libltdl-devel libevent-devel"
+depends="openldap-tools>=${version}_${revision} libevent"
```
[the remark this answers is not in the document]
Comment (Emru1) [reply srcpkgs/openldap/template]: removed from depends, `xbps-query` still shows `libevent` in `run_depends`

File: srcpkgs/python3-ldap/template (hunk 1 above)
Comment (Emru1) [reply srcpkgs/python3-ldap/template]: done

File: srcpkgs/alpine/template [hunk 3]
```diff
@@ -29,7 +29,3 @@ do_build() {
 		CC="$BUILD_CC" CFLAGS="$BUILD_CFLAGS" CPPFLAGS= LDFLAGS="$BUILD_LDFLAGS"
 	make ${makejobs} all CC="$CC"
 }
-
-post_install() {
-	vlicense NOTICE
-}
```
Comment (paper42) [inline srcpkgs/alpine/template:35]: are you sure we can remove this? NOTICE doesn't match Apache-2.0 exactly

**Comment by mhmdanas** [comment]
You may want to close this in favor of https://github.com/void-linux/void-packages/pull/41948.

**Comment by klarasm** [comment]
I just want to say that I did not notice this PR when I created mine (I should probably have looked harder).
@Emru1, I am sorry, I did not want to duplicate your work. If you want to continue here I can close my PR and help with testing this instead.

**Comment by Emru1** [comment]
@klarasm 
no, no, that's okay :)
I am happy that someone picked this up, as I had shortage in my free time recently, so I could not polish this up properly.

**Pull request closed without merging**